elements/rh-tabs/rh-tab-panel.css

```diff
@@ -0,0 +1,21 @@
+#rhds-container {
+  display: block;
+  padding: var(--rh-space-2xl, 32px);
```

Comment:
For vertical box tabs, the panel's padding-block value should be `--rh-space-3xl`.